test/spec/asyncVersioningSpec.js

```diff
@@ -107,25 +111,25 @@ describe("JSONPatchQueue instance", function () {
       ];
 
       beforeEach(function () {
-        obj = {foo: 1, baz: [{qux: 'hello'}]};
+        queue.obj = {foo: 1, baz: [{qux: 'hello'}]};
         //add something to the queue
-        queue.receive(obj, versionedJSONPatch2);
-        queue.receive(obj, versionedJSONPatch3);
+        queue.receive(versionedJSONPatch2);
+        queue.receive(versionedJSONPatch3);
         // receive consecutive patch
-        queue.receive(obj, versionedJSONPatch1);
+        queue.receive(versionedJSONPatch1);
 
       });
 
       it('should apply given JSON Patch sequence', function() {
-        expect(applyPatch).toHaveBeenCalledWith(obj, [{op: 'replace', path: '/baz', value: 'smth'}]);
-        expect(applyPatch.calls.argsFor(0)).toEqual([obj, [{op: 'replace', path: '/baz', value: 'smth'}]]);
+        expect(applyPatch).toHaveBeenCalledWith(queue.obj, [{op: 'replace', path: '/baz', value: 'smth'}]);
```

Comment:
Shouldn't' we still check for `obj`, not to test internal implementation details, but rather the behavior of just `receive`?

Reply:
ping @alshakero